src/library_webgpu.js

```diff
@@ -1375,9 +1375,12 @@ var LibraryWebGPU = {
     '$runtimeKeepalivePush', '$runtimeKeepalivePop',
 #endif
   ],
-  wgpuQueueOnSubmittedWorkDone: function(queueId, callback, userdata) {
+  wgpuQueueOnSubmittedWorkDone: function(queueId, {{{ defineI64Param('signalValue') }}}, callback, userdata) {
     var queue = WebGPU.mgrQueue.get(queueId);
-    return queue["onSubmittedWorkDone"]();
+#if ASSERTIONS
+    assert(signalValue_low === 0 && signalValue_hight === 0, 'signalValue not supported, must be 0');
```

Comment:
Typo: `s/hight/high/`

Reply:
Same for the typo. `return name + '_low, ' + name + '_high';`
Other than that LGTM

Reply:
wow, I tested the assert but since signalValue_low was not 0, signalValue_hight wasn't resolved and didn't throw a ReferenceError. Thanks for catching this!